doc/python/b64.md

```diff
@@ -0,0 +1,198 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.15.2
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.9.0
+  plotly:
+    description: How to format axes of 3d plots in Python with Plotly.
+    display_as: b64
+    language: python
+    layout: base
+    name: b64
+    order: 1
+    page_type: example_index
+    permalink: python/b64/
+    thumbnail: thumbnail/b64.png
+---
+
```

Comment:
Could you add some context on the benefits of doing this. What does it bring for the user?